MLP.py

```diff
@@ -72,21 +72,21 @@ def cv_learn(n_splits, X, y, n_hiddens, layer_size, batch_size, n_ite):
         # Training loop for the specified number of iterations
         for i in range(n_ite):
             # training
-            train_loss = 0
+            subtrain_loss = 0
             for inputs, labels in dataloader:
                 optimizer.zero_grad()
                 loss = loss_func(model(inputs), labels)
                 loss.backward()
                 optimizer.step()
-                train_loss += loss.item()
+                subtrain_loss += loss.item()
```

Comment:
instead of computing loss of each batch and then summing, please compute the loss of both subtrain/validation sets, at the end of every epoch, like we did in class (less noisy than summing loss over batches)